renovate.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": ["github>canonical/renovate-apps"]
+  "extends": ["github>canonical/vanilla-framework"]
```

Comment:
@petermakowski Why changing this? I thought that renovate-apps was our standard for React dependencies?

Reply:
Also, the Vanilla repo doesn't contain an extendable config (see https://github.com/canonical/react-components/pull/997).